iocore/net/SSLUtils.cc

```diff
@@ -1319,6 +1320,12 @@ SSLMultiCertConfigLoader::init_server_ssl_ctx(std::vector<X509 *> &cert_list, co
       SSL_CTX_set_options(ctx, SSL_OP_NO_TICKET);
       Debug("ssl", "ssl session ticket is disabled");
     }
+#endif
+#if OPENSSL_VERSION_NUMBER >= 0x10101000L && !defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
Not a blocker, but we should probably avoid use of OPENSSL_VERSION_NUMBER for new code because it'll be deprecated on OpenSSL 3.0.

Reply:
Ok, replaced with TLS1_3_VERSION